### PR TITLE
Task9

### PR DESCRIPTION
**Pull request by AndreyZelikay**

### CODE REVIEW

**Comment by AndreyZelikay** [comment]
I added pretty simple event listener to this button downward , because it will be more comfortable to review a page. You can just click on it to hide a filtration block.

File: Server/src/main/webapp/resources/js/script.js [hunk 1]
```diff
@@ -360,74 +230,163 @@ class PostCollection {
...
+
+    displayUsersSelect(users) {
+        let menu = document.getElementById('userSelect');
+
+        this._fillPopUpMenu(menu, users);
+    }
+
+    clearSelector(selector) {
+        let last = selector.lastElementChild;
+        while (last && last !== selector.firstElementChild) {
```
Comment (MaksimViarenich) [inline Server/src/main/webapp/resources/js/script.js:334]: Minor. Add empty line

File: Server/src/main/webapp/resources/js/script.js [hunk 2]
```diff
@@ -360,74 +230,163 @@ class PostCollection {
...
+        this._header.querySelector('[class = "my-profile"]').textContent = userName;
+    }
+
+    static displayFiltration() {
+        const filtration = document.getElementById("filtration");
+        if(filtration.style.display !== 'none') {
+            filtration.setAttribute('style', 'display: none');
+        } else {
+            filtration.setAttribute('style', 'display: inline-block');
+        }
```
Comment (MaksimViarenich) [inline Server/src/main/webapp/resources/js/script.js:300]: You can simplify it to
```
var displayStyle = 'display:' + filtration.style.display !== 'none' ? 'none' : 'inline-block';

document.getElementById('filtration').setAttribute('style', displayStyle);
```

File: Server/src/main/webapp/resources/js/script.js [hunk 3]
```diff
@@ -360,74 +230,163 @@ class PostCollection {
...
+        document.getElementById(id).replaceWith(postNode);
+    }
+
+    removePost(id) {
+        document.getElementById(id)?.remove();
+    }
+
+    clearList() {
+        let first = this._postContainer.firstElementChild;
+        while (first && first !== this._postContainer.lastElementChild) {
```
Comment (MaksimViarenich) [inline Server/src/main/webapp/resources/js/script.js:276]: Minor. Add empty line before while

File: Server/src/main/webapp/resources/css/styles.css [hunk 4]
```diff
@@ -379,20 +414,48 @@ footer p {
...
+
+	.button-text {
+		position: relative;
+		bottom: 5px;
+		visibility: visible;
+	}
+}
+
+.tags-select {
+	position: absolute;
```
Comment (MaksimViarenich) [inline Server/src/main/webapp/resources/css/styles.css:433]: Using position absolute can cause a lot of issues) Be careful

File: Server/src/main/webapp/index.html [hunk 5]
```diff
@@ -4,213 +4,49 @@
...
+                <p class = "text" data-target="description"></p>
+                <img class="twit-photo" data-target="photoLink">
+                <p class="name" data-target="author"></p>
+                <p class="tags" data-target="hashTags"></p>
+                <div class="twit-buttons">
+                    <button class="button-delete">Delete</button>
+                    <button class="button-refactor">Refactor</button>
+                </div>
+                <div class="feedback">
+                    <button class = "feedback-button"><img class = "icon" src="upward.png"></button>
```
Comment (MaksimViarenich) [inline Server/src/main/webapp/index.html:44]: ` <button class="feedback-button"></button>`
Make everything consistent. Why in line 38, for example, u write `class="..."` and in line 44 `class = "..."`

**Review by MaksimViarenich — APPROVED** [review]

**Review by MaksimViarenich — APPROVED** [review]

**Pull request merged**